src/components/selectionBoxLayer.ts

```diff
@@ -62,10 +56,12 @@ export module Components {
     public bounds(newBounds: Bounds): SelectionBoxLayer;
     public bounds(newBounds?: Bounds): any {
       if (newBounds == null) {
-        return this._boxBounds;
+        return this._getBounds();
```

Comment:
Can we blow up either here or in the _getBounds when the `propertyMode.VALUE` and no scale.

Thoughts on this?